modules/manage/pages/kubernetes/decommission-brokers.adoc

```diff
@@ -128,7 +128,7 @@ Throughput is the primary measurement required to calculate future data storage
 
 In the example cluster, only six hours of data locally must be retained (any older data is moved to Tiered Storage with a retention of one year). So each broker should have available storage of around 1.2TB (taking into account both throughput and current data).
 
-Cost and use case requirements will dictate how much to spend on local disk capacity. Tiered Storage can help to both decrease costs and expand data retention capabilities. For details, see xref:manage:kubernetes/tiered-storage.adoc[Tiered Storage].
+Cost and use case requirements will dictate how much to spend on local disk capacity. Tiered Storage can help to both decrease costs and expand data retention capabilities. For details, see xref:manage:kubernetes/storage/tiered-storage.adoc[Tiered Storage].
```

Comment:
Cost and use case requirements determine how much to spend on local disk capacity. Tiered Storage can help to both decrease costs and expand data retention capabilities. For details, see xref:manage:kubernetes/storage/tiered-storage.adoc[Tiered Storage].